src/Mvc/samples/MvcSandbox/MvcSandbox.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
```

Comment:
Do you need this? We're not testing runtime compilation with the sandbox

Reply:
We need this just to get the deps file over which is required by the WebApplicationFactory which throws if its not found... :shrug: